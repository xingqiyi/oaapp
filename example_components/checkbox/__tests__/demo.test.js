import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('checkbox');
