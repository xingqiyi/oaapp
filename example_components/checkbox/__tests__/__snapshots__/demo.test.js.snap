// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/checkbox/demo/basic.md correctly 1`] = `
<div>
  <div
    class="am-list"
  >
    <div
      class="am-list-header"
    >
      CheckboxItem demo
    </div>
    <div
      class="am-list-body"
    >
      <div
        class="am-checkbox-item am-list-item am-list-item-middle"
      >
        <div
          class="am-list-thumb"
        >
          <label
            class="am-checkbox-wrapper"
          >
            <span
              class="am-checkbox"
            >
              <input
                class="am-checkbox-input"
                type="checkbox"
              />
              <span
                class="am-checkbox-inner"
              />
            </span>
          </label>
        </div>
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            Ph.D.
          </div>
        </div>
        <div
          class="am-list-ripple"
          style="display:none;"
        />
      </div>
      <div
        class="am-checkbox-item am-list-item am-list-item-middle"
      >
        <div
          class="am-list-thumb"
        >
          <label
            class="am-checkbox-wrapper"
          >
            <span
              class="am-checkbox"
            >
              <input
                class="am-checkbox-input"
                type="checkbox"
              />
              <span
                class="am-checkbox-inner"
              />
            </span>
          </label>
        </div>
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            Bachelor
          </div>
        </div>
        <div
          class="am-list-ripple"
          style="display:none;"
        />
      </div>
      <div
        class="am-checkbox-item am-list-item am-list-item-middle"
      >
        <div
          class="am-list-thumb"
        >
          <label
            class="am-checkbox-wrapper"
          >
            <span
              class="am-checkbox"
            >
              <input
                class="am-checkbox-input"
                type="checkbox"
              />
              <span
                class="am-checkbox-inner"
              />
            </span>
          </label>
        </div>
        <div
          class="am-list-line"
        >
          <div
            class="am-list-content"
          >
            college diploma
          </div>
        </div>
        <div
          class="am-list-ripple"
          style="display:none;"
        />
      </div>
      <div
        class="am-checkbox-item am-checkbox-item-disabled am-list-item am-list-item-middle"
        data-seed="logId"
      >
        <div
          class="am-list-thumb"
        >
          <label
            class="am-checkbox-wrapper"
          >
            <span
              class="am-checkbox am-checkbox-checked am-checkbox-disabled"
            >
              <input
                checked=""
                class="am-checkbox-input"
                disabled=""
                type="checkbox"
              />
              <span
                class="am-checkbox-inner"
              />
            </span>
          </label>
        </div>
        <div
          class="am-list-line am-list-line-multiple"
        >
          <div
            class="am-list-content"
          >
            undergraduate
            <div
              class="am-list-brief"
            >
              Auxiliary text
            </div>
          </div>
        </div>
        <div
          class="am-list-ripple"
          style="display:none;"
        />
      </div>
    </div>
  </div>
  <div
    class="am-flexbox am-flexbox-align-center"
  >
    <div
      class="am-flexbox-item"
    >
      <div
        class="am-checkbox-agree"
        data-seed="logId"
      >
        <label
          class="am-checkbox-agree-label am-checkbox-wrapper"
        >
          <span
            class="am-checkbox"
          >
            <input
              class="am-checkbox-input"
              data-seed="logId"
              type="checkbox"
            />
            <span
              class="am-checkbox-inner"
            />
          </span>
          Agree 
          <a>
            agreement
          </a>
        </label>
      </div>
    </div>
  </div>
</div>
`;
